src/Common/src/Interop/Unix/System.Net.Security.Native/SecuritySafeHandles.cs

```diff
@@ -13,16 +13,65 @@ namespace System.Net.Security
     internal sealed class SafeFreeNegoCredentials : SafeFreeCredentials
     {
         private SafeGssCredHandle _credential;
+        private readonly bool _isNtlm;
+        private readonly string _userName;
+        private readonly bool _isDefault;
 
         public SafeGssCredHandle GssCredential
         {
             get { return _credential; }
         }
 
-        public SafeFreeNegoCredentials(string username, string password, string domain) : base(IntPtr.Zero, true)
+        public bool IsNtlm
```

Comment:
Thanks for the explanation.  Rather than renaming isNtlmOnly and isNtlm to all be isNtlm, could you rename them to be isNtlmOnly?  I can more easily understand the difference between isNtlmOnly and isNtlmUsed than between isNtlm and IsNtlmUsed.  If you could add comments describing what they mean, that'd be helpful, too.